akka-remote/src/main/scala/akka/remote/artery/SystemMessageDelivery.scala

```diff
@@ -42,7 +42,7 @@ private[remote] object SystemMessageDelivery {
   final case class Ack(seqNo: Long, from: UniqueAddress) extends Reply
   final case class Nack(seqNo: Long, from: UniqueAddress) extends Reply
 
-  final case object ClearSystemMessageDelivery
+  final case class ClearSystemMessageDelivery(incarnation: Int)
```

Comment:
A comment describing how this message is used (e.g. is my previous comment correct?) would be nice.